app/models/user.rb

```diff
@@ -1325,7 +1325,7 @@ def subordinate_delegates
   end
 
   private def highest_delegate_role
-    delegate_roles.max_by { |role| role.status_sort_rank }
+    delegate_roles.min_by { |role| role.status_rank }
```

Comment:
This change is confusing. The method is called "highest" but you are computing a minimum.
You should **at least** add a comment, or perhaps even reconsider the whole implementation of `status_rank`.

Reply:
Okay added a comment. Thanks.

Reply:
Why is `status_rank` implemented like this, anyways? In WCA real-world logic (not code), Senior Delegate is considered a "higher" status than Junior Delegate, for example...